IntroducingMethods/src/cst1201/IntroducingMethods.java

```diff
@@ -3,42 +3,88 @@
 import java.util.Scanner;
 
 /**
- * TODO: Description of your class.
+ * Given a string, the program will return the number of words in the string. It
+ * will also then return the first word of that string along with the 'nth' word
+ * of the string.
  *
- * @author TODO: Your name here.
+ * @author TODO: Winnie Wu & Kenny Tsang
  */
 public class IntroducingMethods {
 
     public static void main(String[] args) {
 
-        // TODO: Fill in the body with your code
+        System.out.print("Enter a string: ");
+        Scanner scanner = new Scanner(System.in);
+
+        getInputString(scanner);
     }
 
     /**
-     * Given a Scanner, prompts the user for a String.  If the user enters an empty
-     * String, reports an error message and asks for a non-empty String.  Returns the
-     * String to the calling program.
+     * Given a Scanner, prompts the user for a String. If the user enters an
+     * empty String, reports an error message and asks for a non-empty String.
+     * Returns the String to the calling program.
+     *
      * @param inScanner The Scanner to use to prompt the user.
      * @return The resulting String from the user.
      */
     private static String getInputString(Scanner inScanner) {
```

Comment:
This method is **just** supposed to get an input string, not run the entire program.